camayoc/tests/qcs/api/v1/scans/test_multi_source_scans.py

```diff
@@ -0,0 +1,44 @@
+# coding=utf-8
+"""Tests for ``Scan`` API endpoint for quipucords server.
+
+:caseautomation: automated
+:casecomponent: api
+:caseimportance: high
+:caselevel: integration
+:requirement: Sonar
+:testtype: functional
+:upstream: yes
+"""
+
+import pytest
+
+from camayoc.tests.qcs.api.v1.utils import (
+    prep_all_source_scan,
+    wait_until_state,
+)
+
+
+@pytest.mark.parametrize('scan_type', ['connect', 'inspect'])
+def test_multi_source_create(shared_client, cleanup, scan_type):
+    """Run a scan on a system and confirm it completes.
```

Comment:
So you are trying to scan multiple systems, right? like having multiple sources for different systems or multiple sources pointing to the same system?

Reply:
I'll update this to be clearer